mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -2,8 +2,32 @@ CLASS net/minecraft/class_2874 net/minecraft/world/dimension/DimensionType
 	FIELD field_13073 hasSkyLight Z
 	FIELD field_13077 suffix Ljava/lang/String;
 	FIELD field_20658 biomeAccessType Lnet/minecraft/class_4545;
+	FIELD field_24504 hellish Z
+	FIELD field_24505 dry Z
+	METHOD <init> (ILjava/lang/String;Ljava/lang/String;Ljava/util/function/BiFunction;ZZZLnet/minecraft/class_4545;)V
+		ARG 1 dimensionId
+		ARG 2 suffix
+		ARG 3 saveDir
+		ARG 4 factory
+		ARG 5 hasSkylight
+		ARG 6 hellish
+		ARG 7 dry
```

Comment:
A few stuff here needs updating, like this is now called "ultrawarm" by mojang

Reply:
I called the field "ultraWarm" now